mibipret/data/names_data.py

```diff
@@ -74,16 +75,82 @@
 name_indene = 'indene'
 name_naphthalene = 'naphthalene'
 
-### todo: complete and potentially choose other names
+### Standard names for additional contaminants
+name_styrene = 'styrene'
+name_isopropylbenzene = 'isopropylbenzene'
+name_n_propylbenzene = 'n_propylbenzene'
+name_ethyltoluene = 'ethyltoluene'
+name_2_ethyltoluene = '2_ethyltoluene'
+name_3_ethyltoluene = '3_ethyltoluene'
+name_4_ethyltoluene = '4_ethyltoluene'
+name_trimethylbenzene = 'trimethylbenzene'
+name_123_trimethylbenzene = '123_trimethylbenzene'
+name_124_trimethylbenzene = '124_trimethylbenzene'
+name_135_trimethylbenzene = '135_trimethylbenzene'
+name_4_isopropyltouene = '4_isopropyltouene'
+name_13_diethylbenzene = '13_diethylbenzene'
+name_1245_tetramethylbenzene = '1245_tetramethylbenzene'
+name_2_methylindene = '2_methylindene'
+name_1_methylnaphtalene = '1_methylnaphtalene'
+name_2_methylnaphtalene = '2_methylnaphtalene'
+name_2_ethylnaphtalene = '2_ethylnaphtalene'
+name_16_dimethylnaphtalene = '16_dimethylnaphtalene'
+name_26_dimethylnaphtalene = '26_dimethylnaphtalene'
+
 contaminants = dict(
```

Comment:
There may be some way to get this stuff from a database (or some library that helps with the normalization, although I haven't found anything suitable yet). But for now this is functional and in a clear structure that should be easy to add to.